app/common/data/interfaces/collections.py

```diff
@@ -0,0 +1,32 @@
+from pydantic import UUID4
+from sqlalchemy.exc import IntegrityError
+
+from app.common.data.interfaces.exceptions import DuplicateValueError
+from app.common.data.models import CollectionSchema, Grant, User
+from app.extensions import db
+
+
+def create_collection_schema(name: str, user: User, grant: Grant, version: int = 1) -> CollectionSchema:
```

Comment:
```suggestion
def create_collection_schema(*, name: str, user: User, grant: Grant, version: int = 1) -> CollectionSchema:
```

Can we force kwargs here? I think there's no obvious ordering to the params, so making it clear at the call site might be nicer?